cs-many-resources/MyStack.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+using Pulumi;
+
+class MyStack : Stack
+{
+    [Output]
+    public Output<int> ResourceCount { get; set; }
+
+    [Output]
+    public Output<int> ResourcePayloadBytes { get; set; }
+
+    public MyStack()
+    {
+        int resourceCount = GetEnv("RESOURCE_COUNT", 64);
+        int resourcePayloadBytes = GetEnv("RESOURCE_PAYLOAD_BYTES", 1024);
```

Comment:
Nit: Any reason to use env vars instead of Pulumi config?

Reply:
Good idea, fixed.